pkg/registry/registry.go

```diff
@@ -37,6 +37,8 @@ type Info struct {
 	ServiceName string
 	// Addr will be set in kitex by default
 	Addr net.Addr
+	// SkipListenAddr is used to prevent the listen addr from overriding the Addr
```

Comment:
Put it as last field and  leave a empty line in front of it, because this field is used for judging but not registerInfo